libs/components/datetime/src/lib/modules/date-range-picker/date-range-picker.component.spec.ts

```diff
@@ -210,7 +210,7 @@ describe('Date range picker', function () {
 
     detectChanges();
 
-    selectCalculator(SkyDateRangeCalculatorId.Before);
+    selectCalculator(SkyDateRangeCalculatorId.After);
```

Comment:
This test is checking the start date value, but the `Before` calculator only interacts with the end date. Fixed it.